test/test_prototype_transforms.py

```diff
@@ -1793,22 +1855,27 @@ def test_to_dtype(dtype, expected_dtypes):
             assert transformed_value is value
 
 
+@pytest.mark.parametrize("other_type", [datapoints.Image, datapoints.Video])
+def test_to_dtype_plain_tensor_warning(other_type):
```

Comment:
Could we also check (possibly above) that using `ToDtype()` on one or more Tensors is fine (doesn't warn, properly converts) as long as there's no Image/Video?

I guess it's just a sub-case of the more general `test_simple_tensor_heuristic()`, but to be on the safe side... ?